assets/xml/objects/object_ob.xml

```diff
@@ -1,84 +1,101 @@
+        <Texture Name="gMoonChildOdalwasMaskFaceTex" OutName="moonchild_odalwas_mask_face" Format="ci4" Width="64" Height="64" Offset="0x92C0" />
+        <Texture Name="gMoonChildOdalwasMaskEarTex" OutName="moonchild_odalwas_mask_ear" Format="ci4" Width="64" Height="64" Offset="0x9AC0" />
+        <Texture Name="gMoonChildGyorgsTwinmoldsMaskEyeTex" OutName="moonchild_gyorgs_twinmolds_mask_eye" Format="rgba16" Width="32" Height="32" Offset="0xA2C0" />
+        <Texture Name="gMoonChildGyorgsMaskSkinTex" OutName="moonchild_gyorgs_mask_skin" Format="ci4" Width="64" Height="64" Offset="0xAAC0" />
+        <Texture Name="gMoonChildGyorgsMaskMouthTex" OutName="moonchild_gyorgs_mask_mouth" Format="ci4" Width="64" Height="64" Offset="0xB2C0" />
+        <Texture Name="gMoonChildGyorgsMaskToothHornTex" OutName="moonchild_gyorgs_mask_tooth_horn" Format="ci4" Width="16" Height="16" Offset="0xBAC0" />
+        <Texture Name="gMoonChildGohtsMaskEyeTex" OutName="moonchild_gohts_mask_eye" Format="rgba16" Width="32" Height="64" Offset="0xBB40" />
+        <Texture Name="gMoonChildGohtsMaskTopPatternTex" OutName="moonchild_gohts_mask_top_pattern" Format="rgba16" Width="32" Height="64" Offset="0xCB40" />
+        <Texture Name="gMoonChildGohtsTwinmoldsMaskPatternTex" OutName="moonchild_goht_twinmold_mask_pattern" Format="rgba16" Width="32" Height="64" Offset="0xDB40" />
+        <Texture Name="gMoonChildGohtsMaskSpikeTwinmoldsMaskMandibleTex" OutName="moonchild_gohts_mask_spike_twinmolds_mask_mandible" Format="rgba16" Width="32" Height="32" Offset="0xEB40" />
+        <Texture Name="gMoonChildTwinmoldsMaskSkinTex" OutName="moonchild_twinmolds_mask_skin" Format="ci4" Width="64" Height="64" Offset="0xF340" />
+        <Texture Name="gMoonChildTwinmoldsMaskSnoutTex" OutName="moonchild_twinmolds_mask_snout" Format="ci4" Width="64" Height="64" Offset="0xFB40" />
+
+        <!-- Moonchild Moving Animations -->
+        <Animation Name="gMoonChildWalkingAnim" Offset="0x10880" />
+        <Animation Name="gMoonChildRunningAnim" Offset="0x10CD8" />
+
+        <!-- Moonchild DisplayLists -->
+        <DList Name="gMoonChildPelvisDL" Offset="0x14190" />
+        <DList Name="gMoonChildLeftThighDL" Offset="0x142C0" />
+        <DList Name="gMoonChildLeftLegDL" Offset="0x144D0" />
+        <DList Name="gMoonChildLeftFootDL" Offset="0x14638" />
+        <DList Name="gMoonChildRightThighDL" Offset="0x147A0" />
+        <DList Name="gMoonChildRightLegDL" Offset="0x149B0" />
+        <DList Name="gMoonChildRightFootDL" Offset="0x14B18" />
+        <DList Name="gMoonChildChestBottomDL" Offset="0x14C80" />
+        <DList Name="gMoonChildChestDL" Offset="0x14DA0" />
+        <DList Name="gMoonChildHeadDL" Offset="0x14F18" />
+        <DList Name="gMoonChildLeftArm" Offset="0x15380" />
+        <DList Name="gMoonChildLeftForeArm" Offset="0x154B0" />
```

Comment:
Could you add the `DL` suffix to the missing ones here? There's more below as well